### ISSUE TITLE
Extension cant load avatars in mention menu

### ISSUE THREAD

**Issue by dragosvr92** [Wolfsblvt/mentions#20]
All i see when i mention someone, even if they have an avatar, it cant load any member avatars. 

All i see is their alt message. This is what the broken avatar path looks like. 

< img width="100" height="100" alt="User avatar" src="./../../download/file.php?avatar=65_1264705060.jpg" >

**Comment by Wolfsblvt** [comment]
Can you tell me your board url? Or just the path of the url, is the forum placed inside a subfolder?
May be linked to #7

**Comment by dragosvr92** [comment]
Oh yes. its related to that issue. 
My path is C:\Users\%USERNAME%\Desktop\UniServerZ\www\forums-upgrade

**Comment by Wolfsblvt** [comment]
Ah, so a local path, that's even more complicated :D
I'll check this out with the other issue.

**Comment by diego898** [comment]
Hello,
I am also experiencing this as well. I just installed this extension on a 3.1.5 board. The only other extension I have enabled is advanced bbcode. It can't load users avetar's when there are none, or when there are (the user:diego does have an avatar)

![tagging](https://cloud.githubusercontent.com/assets/742407/9750329/2052f0d2-564b-11e5-8ab7-9a32a60a694e.gif)

**Comment by dragosvr92** [comment]
Same here. It would be best if we could disable the feature all together.

**Comment by prototype2118** [comment]
I'm experiencing the same issue.  I have it working without the auto load right now but if this could be fixed it would make this extension almost perfect.  It's a great way of getting lots of activity on the forums.

**Comment by diego898** [comment]
I think this extension was recently marked abandoned on the phpbb forums :L

**Comment by prototype2118** [comment]
Yes, I saw that too.  I was just hoping that the developer would take a second look at it.  Perhaps someone could fork it as well.  This extension is working really well for me on my forums even without the auto loading feature.  It would just be nice as this was the only bug I encountered.  I think there was some CPU issues as well but I'm probably not experiencing them since I'm not using auto load.

**Comment by bissim** [comment]
Guys, Wolfsblvt got back recently.

**Comment by Wolfsblvt** [comment]
Yeah, still marked as abandoned, but I will continue working on it. Just a bit slow, cause not much free time, but I will be back (:

**Comment by prototype2118** [comment]
This is really great news.  Super glad to hear it.  This is a very excellent extension and has boosted activity on my phpbb forums greatly.

**Comment by dragosvr92** [comment]
Welcome back Wolfsblvt :smile: 
It would help if you could point us on where to go to disable the code that loads avatars.
Or even include a feature that disables avatars in the autocomplete menu.

**Comment by prototype2118** [comment]
@dragosvr92 under the extensions tab in the ACP for phpbb go to the mentions settings and use these.  This is what I use for now and it works well.  It will not auto load the usernames but the mention will still work. 

![mentions_settings](https://cloud.githubusercontent.com/assets/11168324/12984019/ed0acd3c-d0b9-11e5-8c3e-ead3b2cc78e4.PNG)

**Comment by dragosvr92** [comment]
You confused me a bit with autoload instead of autocomplete :stuck_out_tongue: 
I like the autocomplete... I'd just wish the avatar gone. I guess ill just disable it. 
I dislike the bbcodes. The @ mention does the job, and incase we remove the extension, we wont have the broken bbcodes behind.

**Comment by prototype2118** [comment]
@dragosvr92 I do like the autocomplete feature myself but until @Wolfsblvt or someone who can help can fix the avatar bug when phpbb is installed in sub folder then we really have no choice but to disable it until then.

**Comment by dragosvr92** [comment]
True @prototype2118, but i vote to have a feature that completely removes avatars from autocomplete.

**Comment by prototype2118** [comment]
@dragosvr92 I would happen to completely agree but in the meantime this is what I had to do to get it to play nice.  I hope @Wolfsblvt is still contemplating fixing just this bug at least.

**Comment by prototype2118** [comment]
@Wolfsblvt do you think we'll possibly see a fix to this bug sometime soon?

**Comment by kumencz** [comment]
I found this issue too.
I'm using phpBB 3.2.0 and this pull request: Wolfsblvt/mentions#35
I found dirty solution too: revert changes from this commit https://github.com/phpbb/phpbb/pull/2676/commits/31099a8efda204d764a1d6677a80ff5f710f4575 and avatars in autocomplete menu will work.

I'm new to phpBB.

**Comment by Wolfsblvt** [comment]
Thanks for reporting this and for using the project. This project is no longer maintained, so I’m closing all remaining issues as part of archiving the repository. This avoids leaving a false impression that further development or support is planned.

Anyone interested in continuing development is welcome to do so through a fork, subject to the repository’s license and any applicable third-party licenses.
